cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Should this be here?

Reply:
That file came with the new Cypress version, shall I remove it?